<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>810</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Аэропорт</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tbMain">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>921</width>
      <height>271</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tAirport">
     <attribute name="title">
      <string>Аэропорты</string>
     </attribute>
     <widget class="QPushButton" name="btnAddAirport">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>130</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
      <property name="shortcut">
       <string/>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="leAirportInput">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>90</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDeleteAirport">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>180</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>70</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Название</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>30</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Список аэропортов</string>
      </property>
     </widget>
     <widget class="QListWidget" name="lwAirportOutput">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>50</y>
        <width>141</width>
        <height>121</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAirportSearch">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>120</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Поиск</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>70</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Введите название аэропорта</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leAirportSearch">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>90</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tFlight">
     <attribute name="title">
      <string>Рейсы</string>
     </attribute>
     <widget class="QLineEdit" name="leFlightNumberInput">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAddFlight">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="leFlightFromName">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="leFlightToName">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDeleteFlight">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>190</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
     <widget class="QListWidget" name="lwFlightFromName">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>50</y>
        <width>91</width>
        <height>161</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="lwFlightToName">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>50</y>
        <width>91</width>
        <height>161</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Номер</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Название пункта&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>отправления</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>назначения</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Название пункта&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Выберите</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>пункт</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>30</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>отправления</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>30</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>назначения</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Выберите</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>20</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>пункт</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>30</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Список рейсов</string>
      </property>
     </widget>
     <widget class="QListWidget" name="lwFlightAircraft">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>50</y>
        <width>91</width>
        <height>161</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>20</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Выберите</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>30</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>судно</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leFlightAircraft">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Номер судна&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QListWidget" name="lwFlightOutput">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>50</y>
        <width>241</width>
        <height>131</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="leFlightAircraftSearch">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>170</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAddFlightSearch">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>200</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Поиск</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>110</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>назначения</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leFlightToNameSearch">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>130</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="leFlightFromNameSearch">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>80</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="leFlightNumberSearch">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>30</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>60</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>отправления</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>10</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Номер</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>150</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Номер судна&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>50</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Название пункта&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_30">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>100</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Название пункта&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tAircraft">
     <attribute name="title">
      <string>Воздушные суда</string>
     </attribute>
     <widget class="QLineEdit" name="leAircraftModel">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>60</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="leAircraftNumber">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>110</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAddAircraft">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>150</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDeleteAircraft">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>170</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>40</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Модель</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>90</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Номер</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>10</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Список  судов</string>
      </property>
     </widget>
     <widget class="QListWidget" name="lwAircraftOutput">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>30</y>
        <width>221</width>
        <height>131</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAircraftSearch">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>140</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Поиск</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>80</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Номер</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leAircraftModelSearch">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>50</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="leAircraftNumberSearch">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>100</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>30</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Модель</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>961</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tbMain</tabstop>
  <tabstop>leAirportInput</tabstop>
  <tabstop>btnAddAirport</tabstop>
  <tabstop>lwAirportOutput</tabstop>
  <tabstop>btnDeleteAirport</tabstop>
  <tabstop>leFlightNumberInput</tabstop>
  <tabstop>lwFlightFromName</tabstop>
  <tabstop>lwFlightToName</tabstop>
  <tabstop>lwFlightAircraft</tabstop>
  <tabstop>btnAddFlight</tabstop>
  <tabstop>lwFlightOutput</tabstop>
  <tabstop>btnDeleteFlight</tabstop>
  <tabstop>leFlightFromName</tabstop>
  <tabstop>leFlightToName</tabstop>
  <tabstop>leFlightAircraft</tabstop>
  <tabstop>leAircraftModel</tabstop>
  <tabstop>leAircraftNumber</tabstop>
  <tabstop>btnAddAircraft</tabstop>
  <tabstop>lwAircraftOutput</tabstop>
  <tabstop>btnDeleteAircraft</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
